### DISCUSSION TITLE
Merge PRs and Cut Release

### DISCUSSION THREAD

**Discussion by achapkowski** [geomet/geomet#100] [Team Posts]
Hey Maintainers, this library has been pretty quite, but I found some issues with some ESRI JSON representations.  Can we merge the changes and cut a new release?  I don't have access to push to pypi.

**Comment by larsbutler** [comment]
Hey @achapkowski, I'll work on this a bit this weekend. Part of the issue is that the tools for releasing to pypi dropped support for python 2.7, so we'll need to drop support for that as well and release a new version. It's going to be a bit of package work, and a bit of CI/CD work for me to get things working.

**Comment by larsbutler** [comment]
I'm working on this now. Since dropping python version support is a big change, I'm going to release this as a 1.0.0+ version (according to semver guidelines, you should increment the major version number when an API changes; while we're not changing the API, dropping Python support is a breaking change).

**Comment by larsbutler** [comment]
Hi @achapkowski, I've just put up [#84](https://github.com/geomet/geomet/pull/84) to drop old Python versions, add python 3.10 support, and clean up a bunch around testing/package. This will remove a bit of bitrot and allow us to roll new releases.  

Can you please review? 

Once that's merged, let's get your PRs (https://github.com/geomet/geomet/pull/76 and https://github.com/geomet/geomet/pull/82) rebased on the latest `master` and get them through so we can cut a v1.0.0 release.

**Comment by achapkowski** [comment]
ok, i'll review it in a bit.

**Comment by achapkowski** [comment]
I ran the tests and approved the changes.

**Comment by larsbutler** [comment]
Great, thanks! Merged. @achapkowski Can you rebase your open PRs on the latest master? I can help you shape them up and get them reviewed/merged.

**Comment by achapkowski** [comment]
i fixed the one issue you found and rebased @larsbutler

**Comment by larsbutler** [comment]
@achapkowski I added a fix and a test to https://github.com/geomet/geomet/pull/82#pullrequestreview-1145585024. I approved your changes. If you're happy with my additions, go ahead and merge.

**Comment by larsbutler** [comment]
Hey @achapkowski, is there a case which exhibits the SRID bug related to https://github.com/geomet/geomet/pull/76? It would be good to have some tests for this. I'm happy to write the tests if you can give me a sample of an input that causes this.

**Comment by larsbutler** [comment]
Hi all, I've got a patch up with some badly-need documentation updates as preparation for the v1.0.0 release. Could someone review? https://github.com/geomet/geomet/pull/86

Perhaps @tomplex or @achapkowski?
